TESTING.md

```diff
@@ -0,0 +1,34 @@
+# Running Tests
+
+## Prerequisites
+Before running tests, ensure all steps in the [BUILDING.md](https://github.com/apache/geode/blob/develop/BUILDING.md) have been performed.
+
+## Running Tests By Type
+Tests are broken up into three types - unit, integration, and distributed.  They can be executed using the following commands from the Geode repository root directory:
+* Unit tests
+  `./gradlew test --parallel`
+* Integration tests
+  `./gradlew integrationTest --parallel`
+* Distributed tests
+  `./gradlew distributedTest --parallel`
+
+## Running Individual Tests
+To run an individual test, you can either
+1. Run the test in your [IDE](https://github.com/apache/geode/blob/develop/BUILDING.md#setting-up-intellij)
+2. Run the from terminal by specifying the sub-project and test type:
+`./gradlew project:testType --tests testName`
+For example:
+`./gradlew geode-core:test --tests ArrayUtilsTest`
+`./gradlew geode-core:distributedTest --tests ConnectionPoolDUnitTest`
+
+## Running Tests By Category
+To run a specific category of tests (eg: GfshTest):
+`./gradlew -PtestCategory=fullyQualifiedTestClassName testType`
+For example:
+`./gradlew -PtestCategory=org.apache.geode.test.junit.categories.GfshTest distributedTest`
```

Comment:
This example didn't work for me, but I'm feeling like maybe there's something I'm missing.

```
C:\Projects\geode [develop ≡]> ./gradlew.bat -PtestCategory=org.apache.geode.test.junit.categories.GfshTest distributedTest

FAILURE: Build failed with an exception.

* What went wrong:
Task '.apache.geode.test.junit.categories.GfshTest' not found in root project 'geode'.

* Try:
Run gradlew tasks to get a list of available tasks. Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 7s
```

Maybe I was supposed to add the right project first?

Reply:
I was able to reproduce on Windows and have found a working revised command.  Thanks!